packages/bridge/src/runtime/capi.legacy.ts

```diff
@@ -1,5 +1,5 @@
 import { defu } from 'defu'
-import { ComputedRef, computed, getCurrentInstance as getVM, isReactive, isRef, onBeforeMount, onServerPrefetch, reactive, ref, set, shallowRef, toRaw, toRefs, watch } from 'vue'
+import { ComputedRef, computed, defineComponent as defineComponentVue, getCurrentInstance as getVM, isReactive, isRef, onBeforeMount, onServerPrefetch, reactive, ref, set, shallowRef, toRaw, toRefs, watch } from 'vue'
```

Comment:
Could you also change `ComputedRef` to a type only import while you're at it?
```ts
import type { ComputedRef } from 'vue'
```

Reply:
Thanks for the feedback! I got it done. https://github.com/nuxt/bridge/pull/994/commits/7c76d2d9f15e0a3923c90f7a80802fe9c81f23b0

Reply:
@wattanx 
I just realized that there were some missing types in the exports for the legacy Composition API.
I wonder if it would be better to export all the types that @nuxtjs/composition-api exports?
https://github.com/nuxt-community/composition-api/blob/ebae8087366a605d2ddde19d0afce5a04d5e6bc0/src/runtime/composables/vue.ts#L1-L78